src/file_picker.js

```diff
@@ -28,22 +28,22 @@ function allow_showing_hidden(input) {
  * @return {String}      One of files, dirs, or both
  */
 function allow_selecting_files_folders_or_both(input) {
-  const selection = input.dataset.targetFileType;
-  if(! selection) {
-    return 'both';
+  const selection = input.dataset.targetFileType
+  if (!selection) {
+    return 'both'
   }
 
   const match = {
-    'files': 'files',
-    'dirs': 'dirs',
-    'both': 'both'
```

Comment:
That said, about strict - I'm fine with refactors like this. In fact, this is probably the right style to keep.